postgres-appliance/configure_spilo.py

```diff
@@ -251,6 +251,7 @@ def get_placeholders(provider):
     placeholders.setdefault('BACKUP_SCHEDULE', '00 01 * * *')
     placeholders.setdefault('CRONTAB', '[]')
     placeholders.setdefault('PGROOT', os.path.join(placeholders['PGHOME'], 'pgroot'))
+    placeholders.setdefault('WALE_TMPDIR', os.path.abspath(os.path.join(placeholders['PGROOT'], '../tmp')))
```

Comment:
How much space does WAL-E normally consume in the temp dir?  should we be concerned about small disk volumes on the root partition?